forui/lib/src/widgets/button/button_content.dart

```diff
@@ -1,24 +1,32 @@
-part of 'button.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
 
-final class _FButtonContent extends StatelessWidget {
+import 'package:meta/meta.dart';
+import 'package:sugar/sugar.dart';
+
+import 'package:forui/forui.dart';
+
+@internal
+class Content extends StatelessWidget {
```

Comment:
Would this cause a namespace conflict? From my understanding, `@internal` is just a lint warning.

Reply:
It won’t cause a namespace conflict.